apps/api/src/app/bridge/bridge.module.ts

```diff
@@ -18,6 +18,38 @@ import { PreferencesRepository } from '@novu/dal';
 import { SharedModule } from '../shared/shared.module';
 import { BridgeController } from './bridge.controller';
 import { USECASES } from './usecases';
+import { UpsertWorkflowUseCase } from '../workflows-v2/usecases/upsert-workflow/upsert-workflow.usecase';
+import { PostProcessWorkflowUpdate } from '../workflows-v2/usecases/post-process-workflow-update';
+import { PatchStepUsecase } from '../workflows-v2/usecases/patch-step-data';
+import { OverloadContentDataOnWorkflowUseCase } from '../workflows-v2/usecases/overload-content-data';
+import { BuildStepDataUsecase } from '../workflows-v2/usecases/build-step-data';
+import {
+  BuildDefaultPayloadUsecase,
+  CollectPlaceholderWithDefaultsUsecase,
+  PrepareAndValidateContentUsecase,
+  ValidatePlaceholderUsecase,
+} from '../workflows-v2/usecases/validate-content';
+import { BuildAvailableVariableSchemaUsecase } from '../workflows-v2/usecases/build-variable-schema';
+import { ExtractDefaultValuesFromSchemaUsecase } from '../workflows-v2/usecases/extract-default-values-from-schema';
+import { HydrateEmailSchemaUseCase } from '../environments-v1/usecases/output-renderers/hydrate-email-schema.usecase';
+
+const SHARED_USECASES = [
```

Comment:
We should avoid using use cases from another module (which is already happening), as this can lead to tight coupling, complicated maintenance, and circular dependencies. Instead, we need to identify all shared use cases and move them to a separate module.